### PR TITLE
ceph-83573428 creating with exist names

### PR DESCRIPTION
**Pull request by julpark-rh**
pre-requisites:
    1. Create a random subvolume_group name that does not exist
Test operation:
    1. Try to create a subvolume on the subvolume_group name
    2. If the creation fails, memo it
    3. Try to delete a subvolume on the subvolume_group name
    4. If the deletion fails, memo it.
    5. If the creation and deletion are failed, return 0
    
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Y1QYSK/
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573428

Signed-off-by: julpark-rh <yonhyun@gmail.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_volume_with_exist_names.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        if out1 == 0 or out2 == 0 or out3 == 0:
+            return 1
+        log.info("Cleaning up the FS and subvolume and Subvolume group")
+        client1.exec_command(
+            sudo=True, cmd="ceph config set mon mon_allow_pool_delete true"
+        )
+        fs_util.remove_fs(client1, fs_name)
+        return 0
+    except Exception as e:
+        log.info(e)
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_volume_with_exist_names.py:52]: can you change log.info --> log.error in exception block

**Review by hkadam134 — CHANGES_REQUESTED** [review]
Please change it to :
except Exception as e:
        log.error(e)
        log.error(traceback.format_exc())

**Comment by julpark-rh** [comment]
> Please change it to : except Exception as e: log.error(e) log.error(traceback.format_exc())

I have changed those

**Review by hkadam134 — APPROVED** [review]
Looks good to me..

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**